tests/src/core/getExports.js

```diff
@@ -36,11 +44,20 @@ describe('ExportMap', function () {
 
   });
 
-  it('returns a cached copy on subsequent requests', function () {
+  isVisitorKeysSupported && it('returns a cached copy on subsequent requests', function () {
```

Comment:
```suggestion
  (isVisitorKeysSupported ? it : it.skip)('returns a cached copy on subsequent requests', function () {
```
this way it's marked as skipped

Reply:
this is just cherry-picked from #3062 so you should put this as a review on that 🙂